pyEOS/eos.py

```diff
@@ -198,6 +210,29 @@ def replace_config(self, config=None, force=False):
         else:
             raise exceptions.CommandError(result[1]['messages'][0])
 
+    def merge_config(self, config=None):
```

Comment:
I don´t like this for the same reason as "compare_merge_config". However, as they will be supporting soon the config session feature, this method could be handy as we could reuse it for that purpose. Although in that case maybe we want to make it a bit more clear that we are using that feature. Maybe we can call this method just "config_session"?